_posts/2020-04-23-archivesspace-clean-up-an-outline.md

```diff
@@ -0,0 +1,93 @@
+---
+title: ArchivesSpace Clean Up: An Outline
+date: 2020-04-23T16:00:00+00:00
+author: Darren Young
+layout: post
+categories:
+    - Data Cleanup
+    - Collaboration
+tags:
+    - data
+    - description
+    - discovery
+    - learning
+excerpt_separator: <!--more-->
+---
+As Hillel stated in his recent [blog entry](https://blog.rockarch.org/making-connections), the RAC Processing Team has been working on a number of projects to clean up and improve the archival data our institution maintains in order to support the functionality we intend to provide in our discovery and delivery interface. I am writing this post to describe how Amy Berish, Katie Martin, and I have approached deleting and editing a massive collection of data as well to summarize the work we have accomplished to date in collaboration with different colleagues at the RAC. 
+
+<!--more-->
+
+## Realizing the Problems
+
+Facilitating effective discovery and browsing of relationships between archival objects is one of the stated components of [Project Electron’s](https://blog.rockarch.org/setting-sail-the-next-leg-of-project-electron) to improve user experience, and as my colleagues and I have learned, the complex system of relationships that creates and links our archival data can make something as small as a note on an archival item a significant obstacle for archivists and users alike. Speaking for myself, I know that my understanding of archival relationships generally derives from how I describe and interact with them as a processing archivist. The scope of my use of the ArchivesSpace API therefore heavily influences how I perceive different archival objects and the relationships between them. 
```

Comment:
`Facilitating effective discovery and browsing of relationships between archival objects is one of the stated components of Project Electron’s to improve user experience, and as my colleagues and I have learned, the complex system of relationships that creates and links our archival data can make something as small as a note on an archival item a significant obstacle for archivists and users alike.`

This sentence doesn't read right to me, FYI. I think a word might be missing?

Reply:
yep, a word was missing. thanks for catching that!